File: Programmers/Lv.0/C++/글자 이어 붙여 문자열 만들기.cc
// https://school.programmers.co.kr/learn/courses/30/lessons/181915

#include <string>
#include <vector>

using namespace std;

string solution(string my_string, vector<int> index_list) {
  string answer = "";
  for (int i=0; i < index_list.size(); i++) {
    answer += my_string[index_list[i]];
  }
  return answer;
}

File: Programmers/코딩기초트레이닝/JS/글자 이어 붙여 문자열 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181915?language=python3

// 내 답안
function solution(my_string, index_list) {
    var answer = [];
    for (i of index_list) answer.push(my_string[i]);
    return answer.join("");
}
// 더 간결한 답안
function solution(my_string, index_list) {
    return index_list.map((i) => my_string[i]).join("");
}